src/main/resources/locales/cy/translation/hearing-document-upload.json

```
@@ -1,40 +1,40 @@
 {
-  "title": "Welsh translation required",
-  "h1": "Welsh translation required",
-  "headingCaption": "Welsh translation required",
-  "p1": "Welsh translation required",
-  "bullet1": "Welsh translation required",
-  "bullet2": "Welsh translation required",
-  "bullet3": "Welsh translation required",
-  "bullet4": "Welsh translation required",
-  "bullet5": "Welsh translation required",
-  "p2": "Welsh translation required",
-  "p3": "Welsh translation required <a href=\"http://test\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">Welsh translation required</a>",
+  "title": "Uwchlwythwch eich ffeil o ddogfennau ",
+  "h1": "Uwchlwythwch eich ffeil o ddogfennau ",
+  "headingCaption": "Paratoi dogfennau ar gyfer gwrandawiad",
+  "p1": "Ar ôl ichi ddarllen y gorchmynion gan y tribiwnlys, gwnewch yn siŵr:",
+  "bullet1": "mae gennych y set o ddogfennau mewn un ffeil PDF",
+  "bullet2": "os gofynnir am fynegai, bod yr eitemau mewn trefn yn ôl dyddiad neu drefn y mae'r tribiwnlys wedi gofyn amdano",
+  "bullet3": "mae'r eitem mynegai yn cyfeirio at y dudalen gywir yn y ddogfen",
+  "bullet4": "rydych ond yn cynnwys dogfennau a deunydd perthnasol i'ch achos chi",
+  "bullet5": "mae'r ddogfen PDF yn dod o fewn unrhyw derfynau nifer y tudalennau a roddir gan y tribiwnlys",
+  "p2": "Os ydych chi'n cyfeirio at un adran o ddogfen fel contract cyflogaeth, dywedwch pa ran yn y mynegai.",
+  "p3": "Os oes angen i chi gyfeirio at ddogfennau achos blaenorol, ewch i <a href=\"/all-documents?lng=en\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">ddogfennau eich achos (yn agor mewn tab newydd)</a>",
```

If the user is viewing the page in Welsh and clicking on this, they probs shouldn't be taken back to the English version, either changing the language parameter or removing it should be ok as it would be stored in session storage
```suggestion
  "p3": "Os oes angen i chi gyfeirio at ddogfennau achos blaenorol, ewch i <a href=\"/all-documents?lng=cy\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">ddogfennau eich achos (yn agor mewn tab newydd)</a>",
```

```suggestion
  "p3": "Os oes angen i chi gyfeirio at ddogfennau achos blaenorol, ewch i <a href=\"/all-documents\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">ddogfennau eich achos (yn agor mewn tab newydd)</a>",
```